<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="choosePath">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose the Path</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; font-weight:700;&quot;&gt;Git GUI&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="actionsGroup">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>131</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Actions</string>
    </property>
    <widget class="QPushButton" name="addChanges">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Changes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="commitChanges">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Commit Changes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Push Changes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="initRepos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Init</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="pathLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
